tests/patterns/basic_logic/README.md

```diff
@@ -0,0 +1,149 @@
+# Basic Logic Pattern Test Cases
+
+This collection contains fundamental digital logic test cases implemented in Verilog. These test cases serve as examples for RTL to TLV conversion patterns.
+
+## Components
+
+1. **D Flip-Flop** (`d_flipflop.v`)
```

Comment:
This doesn't have much value since the TLV equivalent is an empty file :). TLV methodology disallows async reset.

Reply:
Well... maybe, actually. The Verilog interface should be preserved and in/out hooked up with a 1-cycle delta, so sure.